metadata/src/main/java/org/apache/kafka/controller/PartitionChangeBuilder.java

```diff
@@ -334,11 +334,11 @@ private void tryElection(PartitionChangeRecord record) {
                 targetIsr = List.of(electionResult.node);
                 targetElr = targetElr.stream().filter(replica -> replica != electionResult.node)
                     .collect(Collectors.toList());
-                log.trace("Setting new leader for topicId {}, partition {} to {} using ELR",
-                        topicId, partitionId, electionResult.node);
+                log.info("Setting new leader for topicId {}, partition {} to {} using ELR. Previous partition:{}, change record{}",
```

Comment:
one tiny tiny nit :)
```suggestion
                log.info("Setting new leader for topicId {}, partition {} to {} using ELR. Previous partition:{}, change record: {}",
```

Reply:
Thanks for the suggestion! updated.